src/js/components/AutolinkComponent.jsx

```diff
@@ -0,0 +1,56 @@
+var React = require("react/addons");
+
+/**
```

Comment:
Well, if we had React 0.14 I would probably have included an external package rather than reinventing this wheel. So the comment serves as a reminder, I guess?